src/main/java/com/epam/javacourses2016/task12/SolverTask12.java

```diff
@@ -16,7 +17,7 @@
      * @return Преобразованный список.
      */
     List<Integer> transform(List<Integer> integers, int value) {
-        //TODO
-        return null;
+        Collections.sort(integers);
+        return integers;
```

Comment:
Hardly it will be better, but can be faster. We can reorder numbers by given rule just after first passage, in O(n).